### DISCUSSION TITLE
Back to back favorites

### DISCUSSION THREAD

**Discussion by PhilipCRoberts** [OpenSageTV/Community#76] [Q&A]
I've noticed that if I have 4 back to back favorites, they won't all record even though I have "remove padding on back to back favorites" enabled.  I can get around this by making the non-scheduled show a manual recording.  Has anyone else noticed this?

**Comment by KryptoNyte-Sage** [comment]
I have not noticed that kind of activity with my back-to-back recordings where padding is removed.  Has it always been enabled on your Sage server, or did you just enable it recently, since a restart?

**Reply by PhilipCRoberts** [reply]
I just noticed it recently, but I've used this feature forever.  I did rebuild my system a few months back.

**Comment by eelpout** [comment]
do you see this in the upcoming recording schedule or only when the recordings actually occur?

**Reply by PhilipCRoberts** [reply]
It is in the upcoming schedule.  I noticed it when it missed a recording.

**Reply by PhilipCRoberts** [reply]
This issue is closed.  It was another favorite taking priority.
